simple_rl/tasks/gym/wrappers.py

```diff
@@ -203,21 +203,25 @@ def __init__(self, env, num_stack, lz4_compress=False):
         self.lz4_compress = lz4_compress
 
         self.frames = deque(maxlen=num_stack)
+        #self.realframes = deque(maxlen=num_stack)
 
         low = np.repeat(self.observation_space.low[np.newaxis, ...], num_stack, axis=0)
         high = np.repeat(self.observation_space.high[np.newaxis, ...], num_stack, axis=0)
         self.observation_space = Box(low=low, high=high, dtype=self.observation_space.dtype)
 
     def _get_observation(self):
         assert len(self.frames) == self.num_stack, (len(self.frames), self.num_stack)
-        return LazyFrames(list(self.frames), self.lz4_compress)
+        return (LazyFrames(list(self.frames), self.lz4_compress))
 
     def step(self, action):
         observation, reward, done, info = self.env.step(action)
+        observation = np.divide(observation[0] + observation[1] + observation[2], 3.0)
```

Comment:
The grayscale operation should not happen inside this wrapper. Either we find a Grayscale gym wrapper, or write it as a clean function in this same class